LetsDinnerV2 MessagesExtension/Extensions/UIImage.swift

```diff
@@ -18,4 +18,17 @@ extension UIImage {
         guard let outputImage = filter.outputImage else { return nil }
         return UIImage(ciImage: outputImage)
     }
+    
+    func withColor(_ color: UIColor) -> UIImage {
```

Comment:
Do you just use this to change the color of the image?

Reply:
yes

Reply:
Is this not working?
theImageView.image = theImageView.image?.withRenderingMode(.alwaysTemplate)
theImageView.tintColor = UIColor.red
https://stackoverflow.com/questions/19274789/how-can-i-change-image-tintcolor-in-ios-and-watchkit

Reply:
Both works.